test/qa-tests/buildscripts/resmokeconfig/suites/restore_archive.yml

```diff
@@ -8,9 +8,15 @@ executor:
   js_test:
     config:
       shell_options:
-        eval_prepend: "load('jstests/libs/servers.js'); load('jstests/libs/servers_misc.js');"
```

Comment:
This test suite is only run against 4.4, so we can just use the `servers.js` and `servers_misc.js` files from the `mongo` repo (the default).